PL_FPDF.pkb

```diff
@@ -910,15 +904,15 @@ end p_putstream;
 ----------------------------------------------------------------------------------------
 procedure p_putstream(pData in out NOCOPY blob) is 
 	offset integer := 1;
-    lv_content_length number := dbms_lob.getlength(pdata);
+  lv_content_length number := dbms_lob.getlength(pdata);
 	buf_size integer := 2000;
-	buf raw(2000);
+	buf varchar2(2000);
 begin
 	p_out('stream');
 	-- read the blob and put it in small pieces in a varchar
 	while offset < lv_content_length loop
 	  dbms_lob.read(pData,buf_size,offset,buf);
-	  p_out(utl_raw.cast_to_varchar2(buf), false);
+	  p_out(buf, false);

```

Comment:
It does, DBMS_LOB.READ can also accept VARCHAR2 as the output buffer parameter:
https://docs.oracle.com/cd/B28359_01/appdev.111/b28419/d_lob.htm#i999170